### DISCUSSION TITLE
✨ Manually set Dagger Cloud run title

### DISCUSSION THREAD

**Discussion by stobias123** [dagger/dagger#12424] [Ideas]
### What are you trying to do?

I'd like to be able to set the title of my runs in dagger cloud. I execute multiple pipelines per commit, so it's a little hard to find what I'm looking for based on commit title.

### Why is this important to you?

I run a (sequential) pipeline that 
1. Builds everything
2. Runs terraform for each environment

Right now each of the above is a separate dagger call. It's hard to find say, the terraform dev run when it all looks like this

![image](https://github.com/dagger/dagger/assets/590677/9525d43d-a853-43df-bd4e-16df67a85383)

I imagine a solution could be to just set pipeline labels or something similar

### How are you currently working around this?

Just clicking a whole lot.

**Comment by jedevc** [comment]
cc @marcosnils
